cmd/api/config.go

```diff
@@ -54,13 +54,15 @@ type config struct {
 			Enabled   bool `default:"true"`
 			KeepFiles int  `default:"5"` // number of files to keep
 		}
+		URL string `default:""`
```

Comment:
This is the backup file url. Not sure if it makes sense to have a default value here.